src/SharedPipelineComponents/Datalock/SFA.DAS.CollectionEarnings.DataLock/Application/DataLock/Matcher/MatcherFactory.cs

```diff
@@ -6,12 +6,12 @@ public static IMatcher CreateMatcher()
         {
             return new UlnMatchHandler(
                 new UkprnMatchHandler(
-                    new StandardMatchHandler(
-                        new FrameworkMatchHandler(
-                            new PathwayMatchHandler(
-                                new ProgrammeMatchHandler(
-                                    new PriceMatchHandler(
-                                        new StartDateMatcher(
+                    new StartDateMatcher(
```

Comment:
Does moving the StartDateMatchHandler up, is this going to break tests? and generate alternative DLOCK errors

Reply:
This is how it should have been. If the start date is wrong and there are also other errors it breaks the commitments system